readme.md

```diff
@@ -149,6 +149,26 @@ Levels are as follows:
 2. 256 color support
 3. Truecolor support (16 million colors)
 
+### chalk.wrapper
```

Comment:
This is not a correct heading as it's not available on the `chalk` global, but is rather a constructor option only.

Reply:
What header you think I should use? Maybe just "wrapper" ?
Or I could actually add it to the global `chalk`.